crates/examples/Cargo.toml

```diff
@@ -48,7 +48,7 @@ required-features = ["object/read"]
 
 [[bin]]
 name = "pecopy"
-required-features = ["object/read_core", "object/write_core", "object/pe"]
+required-features = ["object/read_core", "object/write_core", "object/pe", "object/elf", "object/std"]
```

Comment:
This shouldn't require `elf`, right?

Reply:
Thanks, that was accidental. Just pushed a fix to remove it.